src/docs/devices/KinCony-ATF/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: KinCony-ATF (ESP32-S3 SD Card Module)
+date-published: 2024-10-12
+
+type: misc
+standard: global
+board: esp32
+---
+
+![Product](esp32-sd-card-pins.jpg "Product Image")
+
+## Resources
+
+- [ESP32 pin define details](https://www.kincony.com/forum/showthread.php?tid=5515)
+- [YouTube video tour](https://youtu.be/-ujqoLx0BqA)
+
+## ESPHome Configuration
+
+Here is an example YAML configuration for the KinCony-ATF module.
+
+```yaml
+esphome:
+  name: atf
+  friendly_name: atf
+
+esp32:
+  board: esp32dev
+  framework:
+    type: arduino
+
+# Enable logging
+logger:
+
+# Enable Home Assistant API
+api:
+  encryption:
+    key: "w7SHSfiO/jo57FMsWHzLoexB93cf1YmRH7YUVvJuVQ4="
+
+ota:
+  password: "122b54ecaa3db2cc9455e008d112a4b6"
+
+wifi:
+  ssid: "KinCony"
+  password: "a12345678"

```

Comment:
Please remove this value from the file. There should be no passwords or secrets references in any examples. The only exceptions are !secret wifi_ssid and !secret wifi_password.